integration_test/src/test_runner/mod.rs

```diff
@@ -27,6 +27,8 @@ pub enum Error {
         test: String,
         expected: String,
         actual: String,
+        row: usize,
+        column: usize,
```

Comment:
This change here was for my own sanity on fixing the results.

Reply:
Nice, that will be helpful.